.github/workflows/prep-release.yml

```diff
@@ -22,6 +22,7 @@ on:
 jobs:
   prep_release:
     runs-on: ubuntu-latest
+    environment: release
     steps:
       - uses: jupyterlab/maintainer-tools/.github/actions/base-setup@v1
 
```

Comment:
wondering if we need the `actions/create-github-app-token@v1` step in the prep-release workflow?

The example workflows don't seem to have it: https://github.com/jupyter-server/jupyter_releaser/blob/main/example-workflows/prep-release.yml

Reply:
ah indeed, I used to use `GITHUB_ADMIN_TOKEN` in the prep-release step, but actually `GITHUB_TOKEN` is enough

Reply:
I guess we would be fine using the workflows from https://github.com/jupyter-server/jupyter_releaser/blob/main/example-workflows/ directly?

Reply:
you mean importing the workflow from `jupyter-releaser`? I don't know how to do it.

Reply:
Ah no, just copy pasting it.

Reply:
Done